src/API/OnboardingTasks.php

```diff
@@ -776,7 +776,9 @@ public function get_status() {
 	 */
 	public function get_tasks( $request ) {
 		$extended_tasks = $request->get_param( 'extended_tasks' );
-		$lists          = TaskLists::get_lists( $extended_tasks );
+		$lists          = TaskLists::get_lists(
```

Comment:
This was causing an error, as the default `null` value was overriding default parameters in the `TasksLists` class.

Reply:
Nice.  I know this may be pushing the scope of this PR, but I don't think we should be passing anything to `get_lists()`.  It's too specific to the extended task list.

Instead this should be:

```php
TaskLists::maybe_add_extended_tasks( $extended_tasks );
TaskLists::get_lists();
```

Reply:
Agreed, done in https://github.com/woocommerce/woocommerce-admin/pull/7747/commits/0ce596f064371dfc9e5307b98ada708a8fe204b6